presto-hive/src/main/java/com/facebook/presto/hive/HiveLocationService.java

```diff
@@ -57,21 +57,23 @@ public HiveLocationService(HdfsEnvironment hdfsEnvironment)
     @Override
     public LocationHandle forNewTable(SemiTransactionalHiveMetastore metastore, ConnectorSession session, String schemaName, String tableName, boolean tempPathRequired)
     {
-        HdfsContext context = new HdfsContext(session, schemaName, tableName);
+        HdfsContext context = new HdfsContext(session, schemaName, tableName, true);
```

Comment:
This context is used to check the databased (schema) directory. Let's use the `HdfsContext`'s constructor that only takes a schema name (as it is used when creating a schema as well)

Reply:
Commit `Refactor HdfsContext used by HiveLocationService` address all call sites in `HiveLocationService`. This class is not used in FB.